src/syntax/translation/resugar.rs

```diff
@@ -391,6 +391,7 @@ fn resugar_term(term: &core::Term, prec: Prec) -> concrete::Term {
                 vec![resugar_term(arg, Prec::NO_WRAP)], // TODO
             ),
         ),
+        core::Term::Let(ref _scope) => unimplemented!(),//resugar_let(scope, prec),
```

Comment:
Stray comment here - did you want to implement this in this PR, or later? I'm cool if you want to defer it!